app/helpers/application_notifications_helper.rb

```diff
@@ -5,9 +5,26 @@ module ApplicationNotificationsHelper
   # @param [#notification_view_path] notification The target notification
   # @return [String] The view path of the notification
   def notification_view_path(notification)
+    "#{notification_directory_path(notification)}/#{notification.notification_type}"
+  end
+
+  # Returns the view path of the actable type
+  #
+  # @param [Course::Notification] notification The notification object
+  # @param [String] actable_type The lesson plan actable type as a String
+  # @return [String] The view path of the actable type
+  def actable_type_partial_path(notification, actable_type)
```

Comment:
This method is specific to the Consolidated job. Suggest that we take this method out into a separate helper, and included it only for the conslidated job mailer. 

What you will need to do is probably to define a new mailer that inherits from ActivityMailer - that mailer class will be called by the notifier as well.